src/scripts/llm.js

```diff
@@ -32,6 +53,11 @@ async function generateResponse(actor, input) {
         ui.notifications.error("Please set a repetition penalty in the actor sheet.");
         return;
     }
+
+    const { model, tokenizer } = await getModelAndTokenizer(model_path);
```

Comment:
Needs to be called earlier or `let { input_ids } = tokenizer(prompt);` later. That way the variable is defined.